cluster/juju/layers/kubernetes-master/templates/service-always-restart.conf

```diff
@@ -0,0 +1,5 @@
+[Unit]
+StartLimitIntervalSec=0
+
+[Service]
+RestartSec=1
```

Comment:
1 second seems a little spammy, no?

Reply:
Not to me, I like the idea that it'll restart quickly. But this is super subjective so it's hard to have a meaningful debate about it. If you still like 15sec then I'll revert it to that.

Reply:
I went for 10 sec. Cheers.